modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -0,0 +1,117 @@
+
+* provides customized {payg} product images for SLES, SLES for SAP, etc.
+* operates per-region RMT Servers mirroring repositories for products available as {payg}
+
+This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about instance statuses.
+
+Before using {payg} feature make sure that: 
+
+* {productname} server is deployed in the same region as the {payg} instance
+* valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products
+
+
+== Connecting {payg} instance
+
+[[proc-conencting-new-payg]]
+.Procedure: Conencting new {payg}  instance
+[role=procedure]
+. In the {productname} {webui}, navigate  to menu:Admin[Setup Wizard > {payg}], and click btn:[Add {payg}].
+. In the new screen, click btn:[Create] to add new {payg} connection data.
+. In the menu:Add new {payg} data[Description], add the description of the newly created connection.
+. In the menu:Add new {payg} data[Hostname], enter the actual name of the instance as it appears on the cloud.
+. In the menu:Add new {payg} data[Port], enter 22.
+. In the menu:Add new {payg} data[Username], add the actual username as specified in the cloud.
+. In the menu:Add new {payg} data[Password], enter the password.
+. In the menu:Add new {payg} data[Key], add the correct instance key.
+. In the menu:Add new {payg} data[Key Password], add key passphrase.
+
+[IMPORTANT]
+====
+Authentication keys must always be in PEM format.
+====
+
+If you are not connecting directly to the cloud, but via SSH bastion, proceed with <<proc-adding-ssh-bastion-connection-data>>.
+
+Otherwise, continue with <<proc-finishing-payg-connecting>>.
+
+[[proc-adding-ssh-bastion-connection-data]]
+.Procedure: Adding SSH bastion connection data
+[role=procedure]
+. In the menu:Add new {payg} data[Bastion Hostame], add the bastion hostname.
+. In the menu:Add new {payg} data[Bastion Port], add the bastion port number.
+. In the menu:Add new {payg} data[Bastion Username], add bastion the username.
+. In the menu:Add new {payg} data[Bastion Password], add bastion password.
+. In the menu:Add new {payg} data[Bastion Key], add bastion key.
+. In the menu:Add new {payg} data[Bastion Key Password], add bastion key password.
+
+Complete the setup process with with <<proc-finishing-payg-connecting>>.
+
+[[proc-finishing-payg-connecting]]
+.Procedure: Finishing {payg} connecting
+[role=procedure]
+. Click btn:[Create] to complete adding new {payg} connection data
+. You will return to [guimenu]``Admin > Setup Wizard > Pay-as-you-go`` screen.
+. If the authentication data for the instance are correct, the column Status will soon show "Credentials successfully updated".
+
+[IMPORTANT]
+====
+If the invalid data are entered at any point, the newly created instance will be shown in [guimenu]``Admin > Setup Wizard > {payg}``, with column Status displaying error message.
```

Comment:
After creation, the detail about the payg ssh connection data will be displayed. 
The status is also displayed in the payg ssh connection data list.